catalog-info.yaml

```diff
@@ -156,7 +156,7 @@ spec:
         build_branches: true
         build_pull_request_forks: false
         cancel_deleted_branch_builds: true
-        filter_condition: build.branch == "main" || build.branch == "8.15" || build.branch == "8.14" || build.branch == "7.17"
+        filter_condition: build.branch == "main" || build.branch == "8.15" || build.branch == "7.17"
```

Comment:
I just checked https://www.elastic.co/support/eol

Am I correct that we've been building one too many versions forever?